hw/Makefile

```diff
@@ -130,11 +124,12 @@ topgen_rust_pkg: topgen_rust
 
 top: $(tops_gen)
 $(tops_gen): %_gen:
-	${PRJ_DIR}/util/topgen.py -t $(top-hjson) -o ${PRJ_DIR}/hw/$*/ ${toolflags}
+	${PRJ_DIR}/util/topgen.py -t $(top_hjson) -o ${PRJ_DIR}/hw/$*/ ${toolflags}
 
-# This runs cmdgen on all md files of the repo.
+# This runs cmdgen on all md files of the repo using xargs for parallelism
 cmdgen:
-	${PRJ_DIR}/util/cmdgen.py -u '**/*.md'
+	(cd ${PRJ_DIR}; find . -name "*.md" -print0 | \
+	 xargs -0 -i -P 8 -I {} ./util/cmdgen.py -u {})
```

Comment:
Probably, we'd be better off if we could actually identify the outputs, and cmdgen didn't _rewrite_ the source files. Then this wouldn't have to be a phony target and could be parallelized with make's native job handling. But... no reason to let the perfect be the enemy of the good!